source/common/quic/envoy_quic_server_stream.h

```diff
@@ -78,6 +79,8 @@ class EnvoyQuicServerStream : public quic::QuicSpdyServerStreamBase,
   Http::HeaderUtility::HeaderValidationResult
   validateHeader(absl::string_view header_name, absl::string_view header_value) override;
 
+  void enableHttpDatagramSupport();
```

Comment:
I think this definitely deserves a comment to explains what it means to call this method.

Reply:
I added a comment here and above the |http_datagram_handler_| declartion.